src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo MOYO"
+          class="logo__moyo"
+        >
+      </a>
+      <nav class="nav__list">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav__list">
```
don't forget to add space between one-level blocks. Fix it all over the code